pypi_count/command_line_interface.py

```diff
@@ -25,9 +25,12 @@ def main(  # pylint: disable=R0913
     function_with_docstrings: bool = typer.Option(False, "--function_with_docstrings"),
     class_with_docstrings: bool = typer.Option(False, "--class_with_docstrings"),
     class_without_docstrings: bool = typer.Option(False, "--class_without_docstrings"),
+    find_parameters: bool = typer.Option(False, "--find_parameters"),
```

Comment:
The name `find_parameters` sounds like some sort of search like it will return the names of parameters, rather than the number. Could this be renamed to `function_parameters` to be consistent with the other options?

Additionally, this option shouldn't have a default value; if a user wants to find the number of parameters for a specific function, they should always have to specify which function they want the number of parameters for.

Reply:
@caldwella2